base_model_pred.py : on utilise cuda ligne 12 sans vérifier si il est available (on vérifie plus bas seulement, l 45)
probleme : quand on utilise python au lieu du notebook passer par le gpu arrete le processus 'Processus arreté'








résolution : 

-
-
-
-
-
-
-
-https://forum.image.sc/t/unable-to-run-cytomine-algorithms/52393/3 : se connecter au serveur, le relancer.

sudo docker exec -it  software_router bash
pour accéder au répertoire du software rooter (exit pour quitter)
